README.md

```diff
@@ -28,9 +28,17 @@ First add this to your `Cargo.toml`
 
 ```toml
 [dependencies]
-multibase = "0.8"
+multibase = "0.9"
 ```
 
+For `no_std`
+```
+[dependencies]
+multibase = { version ="0.9", default-features = false }
+```
+
+**note**: This crate relies on the [currently unstable](rust-lang/cargo#7915) `host_dep` feature to [compile proc macros with the proper dependencies](https://docs.rs/data-encoding-macro/0.1.10/data_encoding_macro/).
```

Comment:
This should be a full link to https://github.com/rust-lang/cargo/issues/7915

Could you please also add the information that it only works with a nightly version of Rust? "unstable feature" implies that, but that might not be apparent for all users.